examples/models/llama2/export_llama_lib.py

```diff
@@ -304,6 +306,11 @@ def build_args_parser() -> argparse.ArgumentParser:
         action="store_true",
         help="This option is only for coreml, and is only supported for MacOS15+/iOS18+",
     )
+    parser.add_argument(
```

Comment:
Chatted offline, both state and sdpa are iOS18+ feature. Can we do --ios 18 and enable sdpa + stateful model by default?

Reply:
This will be a better UX

Reply:
Addressed in follow up https://github.com/pytorch/executorch/pull/5319